robot_controllers/src/cartesian_twist.cpp

```diff
@@ -282,14 +304,21 @@ void CartesianTwistController::command(const geometry_msgs::Twist::ConstPtr& goa
     return;
   }
 
+  std::string frame;
+  frame = goal->header.frame_id;
```

Comment:
is there actually a need to copy this into a new std::string? -- goal->header.frame_id is already a std::string